### PR TITLE
miRDeep2 scripts do a hard check for an (empty) file named 'install_successful' in the install dir...

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2635.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c27ab31e84554cc597742fb97da804c3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/39bf86928067b56250faf5fdc15e9c24 for a full test report.

File: easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb [hunk 1]
```diff
@@ -20,10 +20,13 @@ dependencies = [
...
 ]
 
-install_cmd = "chmod a+x mirdeep*/src/*.pl && cp -a mirdeep*/src %(installdir)s/bin"
+install_cmd = "cp -a mirdeep*/src %(installdir)s/bin && chmod a+x %(installdir)s/bin/*.pl && "
+
+# scripts include a hard check for a file called 'install_successful' in the install directory...
+install_cmd += "touch %(installdir)s/install_successful"
 
 sanity_check_paths = {
-    'files': ['bin/make_html.pl'],
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb]: this file is gone?

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**